cmd/util/cmd/rollback-executed-height/cmd/rollback_executed_height.go

```diff
@@ -214,7 +223,7 @@ func removeForBlockID(
 		}
 
 		// remove chunkID-blockID index
-		err = headers.RemoveChunkBlockIndexByChunkID(chunkID)
+		err = headers.BatchRemoveChunkBlockIndexByChunkID(chunkID, writeBatch)
```

Comment:
Sometimes, an EN might not have the chunk block index for the given chunk ID. 

If we call `RemoveChunkBlockIndexByChunkID `, then it will return `storage.ErrNotFound` error, and since we want to continue removing other data associated with the block, we handle the `storage.ErrNotFound` specifically, and continue with the next chunk.

How does it work with the BatchWrite? would `BatchRemoveChunkBlockIndexByChunkID` also return NotFound?
It's better to have a test case to understand the batchRemove's behavior when no key is found. 

If `BatchRemoveChunkBlockIndexByChunkID` never returns NotFound error, we could remove the "if" case below.

Reply:
Yes, we will never return `storage.ErrNotFound` in this code, I'm removing the test case